diff --git a/src/reshade_effect_manager.cpp b/src/reshade_effect_manager.cpp
index 1d05cc5..0a33605 100644
--- a/src/reshade_effect_manager.cpp
+++ b/src/reshade_effect_manager.cpp
@@ -56,7 +56,7 @@ static std::string GetLocalUsrDir()
 
 static std::string GetUsrDir()
 {
-    return "/usr";
+    return "@out@";
 }
 
 static LogScope reshade_log("gamescope_reshade");
@@ -1958,4 +1958,4 @@ void reshade_effect_manager_enable_effect()
 void reshade_effect_manager_disable_effect() 
 {
     gamescope_clear_reshade_effect();
-}
\ No newline at end of file
+}
